.travis.yml

```diff
@@ -63,10 +63,7 @@ jobs:
           # Build doxygen if it is not in Travis cache
           if ! [ -x doxygen/build/bin/doxygen ]
           then
-            mkdir -p doxygen/build \
-            && wget http://ftp.stack.nl/pub/users/dimitri/doxygen-${DOXYGEN_VERSION}.src.tar.gz -O- | tar -xz --strip-components=1 --directory doxygen \
-            && ( cd doxygen/build && cmake .. ) \
-            && make -j4 -C doxygen/build
+            scripts/build_doxygen.sh
```

Comment:
You're not calling your script according to its rules shouldn't use be passing in `${DOXYGEN_VERSION}` here?

Reply:
It would have helped if I had typed something along the lines of `git push` after I fixed that problem...  Please could you resolve this conversation before anyone else sees it?  😉